docker/ci/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.20
+FROM golang:1.22
```

Comment:
1.23 is available

Reply:
1.23 will require k8s libs updates and higher kubernetes version. 
For now, I want to update for 1.22 so that dependencies for in our setups won't be broken.

Reply:
@cscetbon Can we merge code about 1.22 go and release casskop? Then we can do a separate release with 1.23